data/standards/manage_OpenStudio_Standards.rb

```diff
@@ -316,14 +336,16 @@ def export_spreadsheet_to_json(spreadsheet_titles, dataset_type: 'os_stds')
 
     # Open workbook
     workbook = RubyXL::Parser.parse(xlsx_path)
+    puts "After parse workbook"
 
     # Find all the template directories that match the search criteria embedded in the spreadsheet title
     dirs = spreadsheet_title.gsub('OpenStudio_Standards-', '').gsub(/\(\w*\)/, '').split('-')
     new_dirs = []
     dirs.each { |d| d == 'ALL' ? new_dirs << '*' : new_dirs << "*#{d}*" }
     glob_string = "#{standards_dir}/#{new_dirs.join('/')}"
     puts "--spreadsheet title embedded search criteria: #{glob_string} yields:"
-    template_dirs = Dir.glob(glob_string).select { |f| File.directory?(f) && !f.include?('data') }
+#    template_dirs = Dir.glob(glob_string).select { |f| File.directory?(f) && !f.include?('data') && !f.include?('prm')}
```

Comment:
why is this commented out?

Reply:
It's been so long I'm not sure. I'm guessing it was added temporarily for debugging. I'll remove the line.